File: IT/Stringhe/Python/hellostring.py
# file hellostring.py

from __future__ import print_function, unicode_literals
from sys import stdout
from os import linesep

# dichiara una stringa di caratteri

s = "Ciao, mondo!"

# stampa la stringa sulla console, carattere per carattere

for c in s:
	stdout.write(c)

# aggiunge un'interruzione di riga

stdout.write(linesep)

File: IT/Stringhe/C++/helloString.cpp
/* file helloString.cpp */

#include <iostream>

using std::cout;
using std::endl;
using std::string;

int main()
{
	/* dichiara una stringa di caratteri */

	string s = "Ciao, mondo!";

	/* stampa la stringa sulla console, carattere per carattere */

	for (int i = 0; i < s.length(); i++) {
		cout << s[i];
	}

	/* aggiunge un'interruzione di riga */

	cout << endl;
}

